src/CMakeLists.txt

```diff
@@ -482,21 +482,29 @@ if(WIN32)
   list(APPEND ceph_common_deps dlfcn_win32)
 endif()
 
-if(WITH_BLUESTORE_PMEM OR WITH_RBD_RWL)
+if(WITH_BLUESTORE_PMEM)
   find_package(ndctl 63 REQUIRED)
   find_package(daxctl 63 REQUIRED)
   if(WITH_SYSTEM_PMDK)
     set(pmdk_COMPONENTS)
-    if(WITH_BLUESTORE_PMEM)
-      list(APPEND pmdk_COMPONENTS pmem)
-    endif()
-    if(WITH_RBD_RWL)
-      list(APPEND pmdk_COMPONENTS pmemobj)
-    endif()
+    list(APPEND pmdk_COMPONENTS pmem)
+    find_package(COMPONENTS ${pmdk_COMPONENTS})
```

Comment:
`<PackageName>` should be the first parameter passed to `find_package()`, so this would fail the build if both `WITH_BLUESTORE_PMEM` and  `WITH_SYSTEM_PMDK` are enabled.